CMakeLists.txt

```
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.11)
```

```suggestion
cmake_minimum_required(VERSION 3.15...3.26)
```

(or 4.0 if you set `PYBIND11_FINDPYTHON ON`)